plugins/filter_kubernetes/kubernetes.c

```diff
@@ -847,6 +847,11 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct flb_kube, kubelet_port),
      "kubelet port to connect with when using kubelet"
     },
+    {
+     FLB_CONFIG_MAP_INT, "kube_token_ttl", "60",
+     0, FLB_TRUE, offsetof(struct flb_kube, kube_token_ttl),
+     "kubelet token ttl"
```

Comment:
This requires a docs PR if it's not already in place - can you link one here?

https://github.com/fluent/fluent-bit-docs/blob/master/pipeline/filters/kubernetes.md

Seems like a really useful bit of functionality to get in

Reply:
documentation PR created: https://github.com/fluent/fluent-bit-docs/pull/649